py/kubeflow/examples/notebook_tests/conftest.py

```diff
@@ -20,6 +20,12 @@ def pytest_addoption(parser):
     "--notebook_path", help=("Path to the testing notebook file, starting from"
                              "the base directory of examples repository."),
     type=str, default="")
+  parser.addoption(
```

Comment:
It looks like test-target-name and the GCS options haven't been added
https://github.com/kubeflow/examples/blob/master/py/kubeflow/examples/notebook_tests/conftest.py